app/models/guest.rb

```diff
@@ -0,0 +1,11 @@
+class Guest < ApplicationRecord
+  has_many :room_guests
+  has_many :rooms, through: :room_guests
+
+  validates_presence_of :name, :nights
+  validates_numericality_of :nights
+
+  def self.total_guests
+    self.distinct.count
```

Comment:
This is ok, but you want to be very careful that this method is implemented as the user story suggests - that it should count the number of guests _this Room_ had. In that case, I would use an instance method in `Room` and use its `guests` association to find the distinct count, as you did here. Otherwise, we'd just be getting every guest from the database, which will always be distinct.